pkg/pillar/utils/cipherutils.go

```diff
@@ -43,15 +54,17 @@ func GetCipherCredentials(agentName string, status types.CipherBlockStatus) (typ
 			cipherBlock.Key(), err)
 		return handleCipherBlockCredError(agentName, cipherBlock, decBlock, err)
 	}
-	if err := proto.Unmarshal(clearBytes, &decBlock); err != nil {
+
+	var zconfigDecBlock zconfig.EncryptionBlock
+	err = proto.Unmarshal(clearBytes, &zconfigDecBlock)
+	if err != nil {
 		log.Errorf("%s, encryption block unmarshall failed, %v\n",
 			cipherBlock.Key(), err)
-		return handleCipherBlockCredError(agentName, cipherBlock, decBlock, err)
-	}
-	if err == nil {
-		log.Infof("%s, cipherblock decryption successful\n",
-			cipherBlock.Key())
+		return handleCipherBlockCredError(agentName, cipherBlock,
+			types.EncryptionBlock{}, err)
```

Comment:
We can pass the already declared decBlock in order to maintain the uniformity of the code.

Reply:
Done.. 
-               return handleCipherBlockCredError(agentName, cipherBlock,
-                       types.EncryptionBlock{}, err)
+               return handleCipherBlockCredError(agentName, cipherBlock, decBlock, err)